backend/src/main/java/de/neuefische/backend/model/ComponentsModelDTO.java

```diff
@@ -20,7 +20,7 @@ public class ComponentsModelDTO {
 
     private String classification;
 
-    private String price;
+    private Double price;
 
 
```

Comment:
In modernen Systemen nicht mehr so relevant, aber ein float würde hier ausreichen und Speicherplatz sparen. 4 Byte vs 8 Byte. Und als Hinweis, falls nicht so gewollt: im ComponentModel ist price noch sein String.